web/src/environments/data-sources/titles.json

```diff
@@ -1,18 +1,16 @@
 {
-    "titles": {
-        "EN": {
-            "0": "Administration",
-            "1": "Custom 1",
-            "2": "Custom 2",
-            "3": "Custom 3",
-            "4": "Custom 4"
-        },
-        "DE": {
-            "0": "Verwaltung",
-            "1": "Gewohnheit 1",
-            "2": "Gewohnheit 2",
-            "3": "Gewohnheit 3",
-            "4": "Gewohnheit 4"
-        }
+    "EN": {
+        "0": "Administration",
+        "1": "Custom 1",
+        "2": "Custom 2",
+        "3": "Custom 3",
+        "4": "Custom 4"
+    },
+    "DE": {
+        "0": "Verwaltung",
+        "1": "Gewohnheit 1",
```

Comment:
hmm, tough one.. if not custom too... i would call it for example "Freifeld" !?

Reply:
or i would prefer this "Zusatzfeld"

Reply:
I like Zusatzfeld